middleware/grpc_middleware_test.go

```diff
@@ -177,8 +178,8 @@ mutators:
 			rules: map[configuration.MatchingStrategy][]rule.Rule{
 				configuration.Regexp: {{
 					Match: &rule.MatchGRPC{
-						Authority:  "<[0-9a-zA-Z]+>.apis.ory.sh",
-						FullMethod: "grpc.testing.TestService/Empty<[a-zA-Z]+>",
+						Authority:  "<.*>",
+						FullMethod: "<.*>Empty<.*>",
```

Comment:
Why did this change?

Reply:
This is closer to the rules I wrote in Cloud, so I tested for it ;)